src/Microsoft.Azure.WebJobs/AutoResolveValue.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Description
+{
+    /// <summary>
+    /// Specify the default behavior for an <see cref="AutoResolveAttribute"/> property when it has an empty value. 
+    /// These only apply if the property is empty. They don't apply if the property is an expression that evaluates to empty. 
+    /// </summary>
+    public enum AutoResolveValue
```

Comment:
it seems strange to me that AppSetting.Default is of string type and AutoResolve.Default is an enum